Tests/SwiftProtobufTests/Test_Reserved.swift

```diff
@@ -17,7 +17,7 @@
 
 import Foundation
 import XCTest
-@testable import SwiftProtobuf
+@testable import SwiftProtobufCore
```

Comment:
I think none of these should change for now. If we do this move without breaking anything all the tests compile without any _real_ change

Reply:
This might work here. Never tried to combination of `@_exported import` and `@_spi` together. We should definitely take a look since `@_spi` has a few more upsides to `@testable` as well like being able to compile in release mode without extra flags. However, I would do this investigation in a follow up since I am not sure which Swift version introduced `@_spi` and if we can actually use it here.